app/views/court_dates/show.html.erb

```diff
@@ -64,3 +64,11 @@ casa_case_path(@casa_case) %></h6>
       class: "btn btn-primary" %>
   </div>
 </div>
+
+<% if current_user.casa_admin? || current_user.supervisor? %>
```

Comment:
anything checking permissions should be in a policy file `if current_user.casa_admin? || current_user.supervisor?`